app/assets/javascripts/sufia_overrides.js

```diff
@@ -0,0 +1,6 @@
+// These variables override those defined in Sufia's uploader.js.
+// application.js makes sure that this file is loaded after Sufia.
+var max_file_size = 4000000000;
+var max_file_size_str = "4 GB";
```

Comment:
Identifier 'max_file_size_str' is not in camel case.<br>'max_file_size_str' is defined but never used.